src/client/datascience/commands/exportCommands.ts

```diff
@@ -125,9 +123,8 @@ export class ExportCommands implements IDisposable {
         interpreter?: PythonEnvironment
     ): IExportQuickPickItem[] {
         const items: IExportQuickPickItem[] = [];
-        const notebook = JSON.parse(contents) as nbformat.INotebookContent;
 
-        if (notebook.metadata && isPythonNotebook(notebook.metadata)) {
+        if (interpreter) {
```

Comment:
What if we don't have the python extension installed?

I think this would be better off being an or with the previous condition.